content/updates/index.json

```
@@ -1,7 +1,7 @@
 {
   "2020-07": {
     "title": "🍦 July Serenity updates",
-    "description": "In July, the rules engine started a makeover to offer you a new screen to more easily create or update rules; a new type of association is introduced to facilitate the creation of packs or sets of products, as well as other improvements on the Connection messages, the spell-check feature and date rules condition!",
+    "description": "In July, the rules engine started a makeover to offer you a new screen to more easily create or update rules; but also check out new association type, as well as other improvements on the Connection messages, the spell-check feature and date rules condition!",
```

```suggestion
    "description": "In July, the rules engine started a makeover to offer you a new screen to more easily create or update rules; but also check out a new association type, as well as other improvements on the Connection messages, the spell-check feature and date rules condition!",
```
J'suis pas sûre.

---

```suggestion
    "description": "In July, the rules engine started a makeover to offer you a new screen to more easily create or update rules; but also check out our new association type, as well as other improvements on the Connection messages, the spell-check feature and date rules condition!",
```
je propose our a la place de a